app/src/main/kotlin/co/netguru/baby/monitor/client/application/ApplicationModule.kt

```diff
@@ -75,6 +78,18 @@ class ApplicationModule {
         OkHttpClient.Builder()
             .build()
 
+    @Provides
+    @Reusable
+    fun provideNotifyBabyCryingUseCase(
```

Comment:
do we need it? since we have `@Inject constructor`

Reply:
Implementation changed  a bit since I need to inject strings from resources, that's why i need custom provider

Reply:
than maybe we can drob `@Inject constructor`?